obsreport/obsreport.go

```diff
@@ -17,10 +17,17 @@ package obsreport // import "go.opentelemetry.io/collector/obsreport"
 import (
 	"go.opentelemetry.io/otel/codes"
 	"go.opentelemetry.io/otel/trace"
+
+	"go.opentelemetry.io/collector/featuregate"
+	"go.opentelemetry.io/collector/internal/obsreportconfig/obsmetrics"
 )
 
 func recordError(span trace.Span, err error) {
 	if err != nil {
 		span.SetStatus(codes.Error, err.Error())
 	}
 }
+
+func InstrumentWithOtel() bool {
```

Comment:
Why public? Who is the user of this?

Reply:
I planned this to be public for all components that want to double instrument with otel-go, but as I've discovered the  `TelemetrySettings` struct, that seems to be a more suitable place, what do you think about that?